wls-basisdaten-service/src/main/resources/db/migrations/h2/V2_0__createHandbuchTable.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE Handbuch
+(
+    wahltagid      varchar(1024) not null,
```

Comment:
```suggestion
    wahltagID      varchar(1024) not null,
```
Wollten wir die nicht einheitlich alle groß schreiben?